package.json

```diff
@@ -44,27 +44,27 @@
     "@aws-sdk/client-secrets-manager": "3.241.0",
     "@aws-sdk/client-ssm": "3.241.0",
     "@aws-sdk/client-sts": "3.241.0",
-    "@fastly/js-compute": "0.7.0",
+    "@fastly/js-compute": "1.0.1",
     "@google-cloud/functions": "2.2.3",
     "@google-cloud/secret-manager": "4.2.0",
     "@google-cloud/storage": "6.8.0",
     "@rollup/plugin-alias": "4.0.2",
-    "@rollup/plugin-commonjs": "23.0.7",
-    "@rollup/plugin-json": "5.0.2",
+    "@rollup/plugin-commonjs": "24.0.0",
+    "@rollup/plugin-json": "6.0.0",
     "@rollup/plugin-node-resolve": "15.0.1",
+    "@rollup/plugin-terser": "0.2.1",
     "archiver": "5.3.1",
     "chalk-template": "0.5.0",
     "constants-browserify": "1.0.0",
     "dotenv": "16.0.3",
     "express": "4.18.2",
     "form-data": "4.0.0",
-    "fs-extra": "10.1.0",
+    "fs-extra": "11.1.0",
     "get-stream": "6.0.1",
     "isomorphic-git": "1.21.0",
     "openwhisk": "3.21.7",
     "proxyquire": "2.1.3",
-    "rollup": "2.79.1",
-    "rollup-plugin-terser": "7.0.2",
```

Comment:
`rollup-plugin-terser` has been replaced by `@rollup/plugin-terser`. All tests are green now, but it'd be great if you could check whether the other dependency updates bear no potential for code breaks.

Reply:
the rollup mode is still experimental and not used in production. so we can merge it. thanks for the fix.